bundles/statistics/statsgrid2016/components/IndicatorParameters.js

```diff
@@ -25,7 +34,53 @@ Oskari.clazz.define('Oskari.statistics.statsgrid.IndicatorParameters', function
         this.container.remove();
         this.container = null;
     },
+    _createUi: function ( datasrc, indId, selections, regionsets, values) {
+        var me = this;
+        var locale = me.instance.getLocalization();
+        var errorService = me.service.getErrorService();
+        var panelLoc = locale.panels.newSearch;
+
+        var cont = jQuery(this.__templates.main());
+        this._anchorEl.append(cont);
```

Comment:
So _anchorEl needs to be set (by indicatorSelected()?) before calling _createUi(), but nothing in this file calls indicatorSelected()? Seems a bit too fragile.